get-forms/index.html

```diff
@@ -3,11 +3,24 @@
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
+    <title>Git forms</title>
     <meta name="description" content="My Github Search form" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
   </head>
   <body>
-    <form action="https://github.com/search" method="get"></form>
+    <form action="https://github.com/search" method="get">
+      <input type="search" placeholder="search" name="q" />
+      <label>star</label>
+      <select name="s">
+        <option name="stars">stars</option>
```

Comment:
first option can be "Type" with no name value so it can guide the user and doesn't effect the page